src/resources/postcss/tribe-events-single-blocks-reskin.pcss

```diff
@@ -0,0 +1,51 @@
+/**
+ * Block editor event single v2 reskin
+ *
+ * @since TBD
+ */
+
+/* = Website block view styles
+============================================= */
+.tribe-block__event-website {
+	padding: 20px 0;
+
+	a {
+		background-color: var(--tec-color-link-accent);
+		border: none;
+		border-radius: 4px;
+		font-size: 1.125rem;
+		font-weight: var(--tec-font-weight-bold);
+		min-height: 40px;
+		padding: 0 20px;
+	}
+}
+
+#tribe-events-content .tribe-block__event-website a {
+	color: var(--tec-color-text-events-bar-submit-button);
+
+	&:hover,
+	&:focus {
+		color: var(--tec-color-text-events-bar-submit-button-hover);
+	}
+
+	&:active {
+		color: var(--tec-color-text-events-bar-submit-button-active);
+	}
```

Comment:
This variable is meant for the events bar, and can be overridden using the customizer. Use the background color instead. Also, it seems this variable uses the same color for hover and active states, so might as well just do something like:

```
#tribe-events-content .tribe-block__event-website a {

    &,
    &:hover,
    &:focus,
    &:active {
        // styles
    }
}
```

Reply:
Updated.